src/pmdas/resctrl/GNUmakefile

```diff
@@ -49,6 +49,7 @@ install_pcp install:	default
 	$(INSTALL) -m 755 -d $(PMDATMPDIR)
 	$(INSTALL) -m 755 -t $(PMDATMPDIR) Install Remove $(CMDTARGET) $(LIBTARGET) $(PMDAADMDIR)
 	$(INSTALL) -m 644 -t $(PMDATMPDIR) root pmns domain.h $(DFILES) $(PMDAADMDIR)
+	$(INSTALL) -m 644 sys-fs-resctrl.mount $(PCP_ETC_DIR)/systemd/system/sys-fs-resctrl.mount
```

Comment:
@stanfordcox looking more closely, I think this should be installing to $(PCP_SYSTEMDUNIT_DIR) ... there's lots of mount files in there (I don't have any below the /etc/systemd/system directory locally here)

Reply:
Ah, I was mimicing the way the blog described the mount process and location.  I see what you mean though and will give that a try.

Reply:
Ah so distribution files go into /usr/lib/systemd/. Modifications done by system administrator go into /etc/systemd/system/